enhancements/update/standalone-worker-surge.md

```diff
@@ -0,0 +1,295 @@
+---
+title: machine-config-pool-update-surge-and-nodedraintimeout
+authors:
+  - jupierce
+reviewers: 
+  - TBD
+approvers: 
+  - TBD
+api-approvers: 
+  - TBD
+creation-date: 2024-04-29
+last-updated: 2024-04-29
+tracking-link:
+  - TBD
+see-also:
+  - https://github.com/openshift/enhancements/pull/1571
+---
+
+# MachineConfigPool Update Surge and NodeDrainTimeout
+
+## Summary
+
+Add `MaxSurge` and `NodeDrainTimeout` semantics to `MachineConfigPool` to improve the predictability 

```

Comment:
You can find these values in CAPI MachineDeployment today: https://github.com/Nordix/cluster-api/blob/9a2d8cdc5ad681ba407e47106cb159bfa708763c/config/crd/bases/cluster.x-k8s.io_machinedeployments.yaml . 
I'm assuming we will not be suggesting direct manipulation of `MachineDeployment` once we are on CAPI; i.e. we will still be using `MachineConfigPool` as the abstraction / customer touch point for this type of configuration.

Reply:
> I'm assuming we will not be suggesting direct manipulation of MachineDeployment once we are on CAPI; i.e. we will still be using MachineConfigPool as the abstraction / customer touch point for this type of configuration.

I disagree. We are expecting users to move towards MachineDeployment instead of MachineSets to provide greater abstractions and upgrade features such as RollingUpdates. MachineConfigPool is really about the software that is configured on a group of nodes, the MachineDeployment IMO is the one that specifies how that gets onto the nodes. Whether that is in-place or a rolling update